department-of-veterans-affairs/vets-website
# [18187] VVC Video Appointment Details

## Description
This PR adds updated VVC appointment details page styling and interactions.

How to test

1. Make sure the `va_online_scheduling_homepage_refresh` flag is turned on
2. On the appointments page `http://localhost:3001/health-care/schedule-view-va-appointments/appointments/` Click on each video appointment type: [Atlas, VA Location, At Home, VA Device]
3. Verify each matches the view in the specs

## Testing done
Local and unit

## Screenshots
![image](https://user-images.githubusercontent.com/8315447/106800166-1f9f6280-662e-11eb-95f4-dc57a3f6651f.png)
![image](https://user-images.githubusercontent.com/8315447/106800208-2a59f780-662e-11eb-838f-29eddab41e96.png)
![image](https://user-images.githubusercontent.com/8315447/106800243-33e35f80-662e-11eb-87f3-d95dd136b437.png)
![image](https://user-images.githubusercontent.com/8315447/106800268-3cd43100-662e-11eb-9b31-30fd69b1d705.png)
![image](https://user-images.githubusercontent.com/8315447/106800303-49588980-662e-11eb-8cd8-651a984deaa1.png)
![image](https://user-images.githubusercontent.com/8315447/106800351-570e0f00-662e-11eb-8acd-11cb243a35c6.png)

## Acceptance criteria
- [ ] Changes are behind the va_online_scheduling_homepage_refresh feature flag
- [ ] When user clicks/taps a VVC appointment card, appointment details open on a new page
- [ ] Appointment detail page has a unique URL
- [ ] DESIGN - All desktop, tablet, and mobile designs match the specs
Mobile confirmed - https://www.sketch.com/s/4198dca5-69c7-4769-9686-d213c900876d/a/Kv9bDLV#Inspector
Mobile expanded - https://www.sketch.com/s/4198dca5-69c7-4769-9686-d213c900876d/a/wL9poM9#Inspector
All fields template - https://www.sketch.com/s/4198dca5-69c7-4769-9686-d213c900876d/a/ELn2Ldx#Inspector

## Definition of done
